libs/libcommon/src/libcommon/processing_graph.py

```diff
@@ -32,7 +32,7 @@ class ProcessingStep:
     - the next steps (the steps which previous step is the current one)
     """
 
-    endpoint: str
+    name: str
```

Comment:
One question: we add no deprecation for the `endpoint` parameter?

We used to do it in `datasets` and we already did it for `max_size_fallback` in `workers/first_rows.py`.
However, I'm wondering if it is worth it...

Reply:
Not sure, in this case we want to totally avoid using endpoint as part of processing steps (Separate that logic)

Reply:
Yes, I think we can break the compatibility, as the processing graph is hard-coded (cannot be set up in the env vars).

But, we should write it in the next release notes

Reply:
I moved endpoint to a property with deprecation warning